sycl/test/matrix/matrix-tf32-test.cpp

```diff
@@ -0,0 +1,165 @@
+// RUN: %clangxx -fsycl -O2 %s -o %t.out
+
+#include <CL/sycl.hpp>
+#if (SYCL_EXT_ONEAPI_MATRIX == 2)
+#include <iostream>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental::matrix;
+
+#define SG_SZ 8
```

Comment:
I have a macro indigestion. :-) Please use `auto constexpr` for example

Reply:
That is a good use case. But I guess in that case you would have a `#ifndef SG_SZ` around this.